egs/librispeech/s5/local/chain/run_chain_common.sh

```diff
@@ -3,133 +3,76 @@
 # this script has common stages shared across librispeech chain recipes
 set -e
 
-# configs for 'chain'
-stage=0
-# chain options
-frames_per_eg=150
-max_wer=
+stage=11
+
+# input directory names
+gmm_dir=
+ali_dir=
+lores_train_data_dir=
 
 # output directory names
-dir=
-treedir=
 lang=
-min_seg_len=
+lat_dir=
+treedir=
```

Comment:
for the sake of consistency with the rest of the variable names in this script please modify `treedir` to `tree_dir`.